### PR TITLE
Add Conference Program 

### PR DESCRIPTION
**Pull request by Megmugure**

### CODE REVIEW

**Review by Fnyasimi — COMMENTED** [review]
@bkmgit  will this format of the programme render well on the page?

File: pages/en/committee.markdown [hunk 1]
```diff
@@ -19,7 +19,7 @@ Pauline Karega is a bioinformatics student and researcher working on evolutionar
 ## Margaret Wanjiku
 ![Add Profile Photo](https://sbcf.fr/wp-content/uploads/2018/03/sbcf-default-avatar.png)
 
-<!--add bio here-->
+Margaret serves the role of secretary to the BHKI steering committee. She is a bioinformatician with a keen interest in omics and big data analysis.
```
Comment (Fnyasimi) [inline pages/en/committee.markdown:22]: @Megmugure  this same info is in the #18 pull? Maybe incorporate the suggestions from @bkmgit

File: pages/en/index.markdown [hunk 2]
```diff
@@ -26,26 +26,71 @@ interact with other researchers and more.
...
+|14:20|14:50|Keynote 1: Open science| | |Chair:|
+|14:50|15:00|Break| | | |
+|15:00|15:55|Reproducible Research| | |Chair:|
+|15:15| |Research Data Mangement| | | |
+|15:30| |Project Planning and Management| | | |
+|15:45| |Q&A| | | |
+|15:55|16:00|Break| | | |
+|16:00|17:00|Workflow Management Systems| | |Chair:|
+|16:15| |Open source software/ tools| | | |
+|16:30| |Ethical aspcets in data science| | | |
```
Comment (bkmgit) [inline pages/en/index.markdown:45]:
```suggestion
|16:30| |Ethical aspects in data science| | | |
```
Spelling correction

**Comment by bkmgit** [comment]
> @bkmgit will this format of the program render well on the page?

- Roughly seems ok. Can fix if needed. 
- Should there be spaces for contributed talks? It is not clear where there are speakers, or which topics speakers should submit talk proposals. 
-  Longer breaks maybe helpful, 10 minutes instead of 5 minutes.

File: pages/en/committee.markdown (hunk 1 above)
Comment (Megmugure) [reply pages/en/committee.markdown:22]: I accidentally committed it twice, I was afraid that there might be some conflicts. I hope it can be rectified.

Comment (bkmgit) [reply pages/en/committee.markdown:22]: Can be fixed when merging, so should be ok.

**Comment by bkmgit** [comment]
Adding tentative program is helpful. Perhaps leave theme titles for each day and then have a bulleted list with topics of interest. For example day 4 could be:
## Day 4, Friday, April 29, 2022
|Start (EAT)|End (EAT)|Session 4: Unconference (open panel discussions)|Title|Presenter|Chair:|
|:---:|:---:|:---:|:---:|:---:|:---:|
|14:00|14:50|Session 1| | |Chair:|
|14:55|15:00|Break| | | |
|15:00|15:50|Session 2| | |Chair:|
|15:55|16:00|Break| | | |
|16:00|17:00|Session 3| | |Chair:|
|17:00|18:00|Networking hour| | | |
| | | | | | |

Above this, there could be a section "Conference Themes"
- Open Science
  - Reproducible Research
  - Research Data Management
  - Project Planning and Management
  - Workflow Management Systems
  - Open source software/ tools 
  - Ethical aspects in data science 
- Genomics
  - Pathogen Genomics
  - Human/ Animal Genomics
  - Plant Genomics
  - Evolutionary Genomics
- One Health
  - Antibiotic-resistant germs
  - Vector-borne diseases
  - Diseases in food animals
  - Human-animal bond
  - Contamination of water
- Unconference
  - Careers in open science
  - Alternative career paths in genomics
  - Working and studying outside Kenya

**Comment by Megmugure** [comment]
> Adding tentative program is helpful. Perhaps leave theme titles for each day and then have a bulleted list with topics of interest. For example day 4 could be:
> ## Day 4, Friday, April 29, 2022
> Start (EAT) 	End (EAT) 	Session 4: Unconference (open panel discussions) 	Title 	Presenter 	Chair:
> 14:00 	14:50 	Session 1 			Chair:
> 14:55 	15:00 	Break 			
> 15:00 	15:50 	Session 2 			Chair:
> 15:55 	16:00 	Break 			
> 16:00 	17:00 	Session 3 			Chair:
> 17:00 	18:00 	Networking hour 			
> 					
> 
> Above this, there could be a section "Conference Themes"
> 
>     * Open Science
>       
>       * Reproducible Research
>       * Research Data Management
>       * Project Planning and Management
>       * Workflow Management Systems
>       * Open source software/ tools
>       * Ethical aspects in data science
> 
>     * Genomics
>       
>       * Pathogen Genomics
>       * Human/ Animal Genomics
>       * Plant Genomics
>       * Evolutionary Genomics
> 
>     * One Health
>       
>       * Antibiotic-resistant germs
>       * Vector-borne diseases
>       * Diseases in food animals
>       * Human-animal bond
>       * Contamination of water
> 
>     * Unconference
>       
>       * Careers in open science
>       * Alternative career paths in genomics
>       * Working and studying outside Kenya

Updated

**Comment by bkmgit** [comment]
Seems ok to me.

**Pull request merged**